File: app/src/main/java/com/jwu5/giphyapp/model/Pagination.java
package com.jwu5.giphyapp.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Jiawei on 8/17/2017.
 */
public class Pagination {
    @SerializedName("count")
    @Expose
    private Integer count;
    @SerializedName("offset")
    @Expose
    private Integer offset;

    public Integer getCount() {
        return count;
    }

    public void setCount(Integer count) {
        this.count = count;
    }

    public Integer getOffset() {
        return offset;
    }

    public void setOffset(Integer offset) {
        this.offset = offset;
    }

}

File: app/src/main/java/com/jwu5/giphyapp/network/model/Pagination.kt
package com.jwu5.giphyapp.network.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Jiawei on 8/17/2017.
 */
class Pagination {
    @SerializedName("count")
    @Expose
    var count: Int? = null
    @SerializedName("offset")
    @Expose
    var offset: Int? = null

}
